client/store.js

```diff
@@ -134,10 +147,20 @@ const category = (state = { categories: [] }, action) => {
   }
 };
 
+const review = (state = { reviews: [] }, action) => {
+  switch (action.type) {
+    case SET_REVIEWS:
+      return { ...state, reviews: action.reviews };
```

Comment:
Here we are setting all of the reviews from the db onto state.  (Yes, I know, looks very different than when have one reducer.  I did this for the other reducers and now state is showing properly.)
```suggestion
      return action.reviews ;
```